### ISSUE TITLE
Multi Part Series Navigation Fails Lighthouse Accessibility test.

### ISSUE THREAD

**Issue by Jieiku** [Jieiku/abridge#202]
I performed a lighthouse scan of this page in desktop mode:

https://abridge.pages.dev/overview-abridge/

Throws an issue because there is something other than `<li>` items within the list.
In this case the `<details><summary>More ...</summary></details>`

![2024-08-22_09-19-53](https://github.com/user-attachments/assets/c224e34b-2ff1-4872-abcc-e97ff91312d6)

**Comment by Hysterelius** [comment]
You could fix it with:
```html
</li>
<series>
....
</series>
<li>
```

**Comment by Jieiku** [comment]
I tried playing around with the nesting, but could not make google lighthouse happy, so I made it so that it is no longer a list:

https://github.com/Jieiku/abridge/commit/9e052f0a552497be4265ab2ca3a977f6cc15b224
